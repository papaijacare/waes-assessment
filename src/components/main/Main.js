import React from 'react'
import { Container } from 'semantic-ui-react'

import Header from './Header'
import Grid from './Grid'

import 'semantic-ui-css/semantic.min.css'

const Main = () => (
  <Container>
    <Header />
    <Grid />
  </Container>
)
  
export default Main