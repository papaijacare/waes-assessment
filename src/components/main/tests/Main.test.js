import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Main from '../Main'

describe('Main', () => {
    test('renders correctly', () => {
        const wrapper = shallow(<Main />)

        expect(toJson(wrapper)).toMatchSnapshot()
    })    
})