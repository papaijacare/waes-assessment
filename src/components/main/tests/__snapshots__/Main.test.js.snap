// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders correctly 1`] = `
<Container>
  <Component />
  <Component />
</Container>
`;
